common/featureset.cpp

```diff
@@ -21,6 +21,7 @@ void FeatureSet::initalizeFeatureList(QMap<QString, bool> &featureList) {
     featureList.insert("client_warnings", false);
     featureList.insert("mod_log_lookup", false);
     featureList.insert("client_inactivetimeout", false);
+    featureList.insert("donator_recognition", false);
```

Comment:
I'm still not convinced it does - the worst that happens is that the client will get a 'user limit reached' rejection and be unable to properly display it. But that code path now has a bit of text saying something like 'this probably means your client is out of date and needs to be updated'